session.go

```diff
@@ -2265,6 +2265,10 @@ type inflightCachedEntry struct {
 // not be used on production systems with very high load.
 type Tracer interface {
 	Trace(traceId []byte)
+	AllTraceIDs() [][]byte
```

Comment:
You should not populate these methods here.
Only method gocql needs is `Trace(traceId []byte)`

Reply:
but If I create tracer using NewTracer it returns instance of Tracer and I am not able to perform these functions e.g. AllTraceIDs

Reply:
I discovered that when I was trying to replace NewTraceWriter with NewTracer in tablet integration tests

Reply:
@dkropachev When I try to return `*Tracer` I have an error: `cannot use &tracer{…} (value of type *tracer) as *Tracer value in return statement: *tracer does not implement *Tracer (type *Tracer is pointer to interface, not interface)`

Reply:
I have pushed commit, please take a look.